include/CLine3D.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "TVector3.h"
+
+class CLine3D
+{
+ private:
+  TVector3 _point;
+  TVector3 _direction;
+  TVector3 _u;
+  TVector3 _v;
+
+ public:
+  CLine3D() {};
+  CLine3D(const TVector3 &p, const TVector3 &d);
```

Comment:
I know it's a matter of personal preference, but as a C++ programmer, I would emphasize types over syntax. 

`Line3D(const TVector3& p, const TVector3& d)`

Or even better, in my opinion, to make it more readable (to be read in reverse order as _p is a const reference to a TVector3_ ):

`Line3D(TVector3 const& p, TVector3 const& d)`

https://isocpp.org/wiki/faq/style-and-techniques#whitespace